Makefile

```diff
@@ -0,0 +1,371 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
I don't think this file should be included. Could you add it to your .gitignore?